src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__content">
+    <article class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        class="card__image"
+        alt="iMac 27 retina picture"
+      >
+      </img>
+
+      <h3 class="card__description">
+        APPLE A1419 iMac 27&#34; Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="card__reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__reviews-text">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-label">Price:</p>
+        <p class="card__price-price">$2,199</p>
+      </div>
+
+      <div class="card__button">
+        <a
+        data-qa="hover"
+        class="card__button-link"
+        href="#buy"
+      >
+        BUY
+        </a>
+      </div>
```

Comment:
Why do you need extra div?
Also, transform text using CSS.
```suggestion
      <div class="card__button">
        <a
          data-qa="hover"
          class="card__button-link"
          href="#buy"
        >
          buy
        </a>
      </div>
```